src/logic/wallets/getWeb3.ts

```diff
@@ -62,7 +62,7 @@ const isHardwareWallet = (walletName: string) =>
 const isSmartContractWallet = async (web3Provider: Web3, account: string): Promise<boolean> => {
   const contractCode = await web3Provider.eth.getCode(account)
 
-  return contractCode.replace(EMPTY_DATA, '').replace(/0/g, '') !== ''
+  return !!contractCode && contractCode.replace(EMPTY_DATA, '').replace(/0/g, '') !== ''
```

Comment:
I'm wondering how the return value can even be undefined. According to the [docs](https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html#getcode), it's always a string.

Anyway, this should definitely fix the exception from Sentry :+1:

Reply:
yes, its strange